src/opam_file.mli

```diff
@@ -57,6 +57,12 @@ module Mutator : sig
       file it is inserted at the top of the file. *)
   val set_list : string -> ('a -> value) -> 'a list -> t
 
+  module No_overwrite : sig
```

Comment:
Do we still need this Mutator module? It seems a bit complicated for the job now. All we need now is a function to create a list of opam fields from a project+package and filter out any field that is already in the template.

Reply:
I got rid of it.